### PR TITLE
fix: `mstx_balance` to `ustx_balance` in tomls and readmes

### PR DESCRIPTION
**Pull request by zone117x**
Closes https://github.com/blockstack/stacks-blockchain/issues/2151

We changed a config variable name from `mstx_balance` to `ustx_balance` in a previous PR. This PR fixes all the example tomls and readme references. 

For backwards compatibility, the `mstx_balance` option was fixed so that it still works but prints a deprecation notice -- added this after report of sudden broken balance config.

### CODE REVIEW

File: testnet/stacks-node/src/config.rs [hunk 1]
```diff
@@ -30,17 +29,28 @@ pub struct ConfigFile {
     pub block_limit: Option<BlockLimitFile>,
 }
 
+#[derive(Clone, Deserialize, Default)]
+pub struct LegacyMstxConfigFile {
```
Comment (jcnelson) [inline testnet/stacks-node/src/config.rs:33]: I don't think we need to worry about legacy support -- we're pre-release still.  Maybe we can just do a `panic!()` if we see `mstx_balance` so users know to change it to `ustx_balance`.

**Review by jcnelson — APPROVED** [review]
LGTM!

**Review by lgalabru — APPROVED** [review]
Looks good to me! cc @wileyj @CharlieC3 for the devops repo

File: testnet/stacks-node/src/config.rs (hunk 1 above)
Comment (zone117x) [reply testnet/stacks-node/src/config.rs:33]: I think we should leave it for now to make things easier for devs. We can remove after next marge to master.

**Review by kantai — APPROVED** [review]
This LGTM!

I think we should keep the backwards compatibility path in place for now, it makes people's lives easier who already have miners running, and we can always remove the path from the next version.

**Pull request merged**